### ISSUE TITLE
Works only after minimising the app and opening it back

### ISSUE THREAD

**Issue by git-anshuls** [react-native-google-cast/react-native-google-cast#172]
Description:
The application fails to detect any nearby device when it started, but when I minimize it and open it again, it works perfectly fine.

- RNGC Library Version: 3.2.2
- OS: Android (Have tested on multiple versions of android OS)

1. Open the application.
2. Go to the page where Chromecast is used.
3. The Chromecast icon does not show up, even if the device is there in the network.
4. minimize the app, and open it back.
5. works perfectly fine.

**Expected behavior**
The Chromecast icon should be visible.

**Comment by alejandrorangel** [comment]
@git-anshuls  I was expiring the same behavior, I force my app to use `com.google.android.gms:play-services-cast-framework:16.2.0` which is the version that was being used before and everything when back to normal. hopefully, that would work for you

**Comment by hapo-thinhnh** [comment]
@git-anshuls @alejandrorangel you have solved this problem yet? I am having the same problem, help me please.

**Comment by petrbela** [comment]
I think this was caused by not initializing the CastContext in MainApplication. Please make sure to read the updated setup documentation. If you still experience problems, please open a new issue.
